### ISSUE TITLE
Conflict resolution - is it poasible to paint positive picture 

### ISSUE THREAD

**Issue by Sabryr** [coderefinery/git-intro#130]
Conflict resolution is shown as a scary thing and gives the impression that this is bad. There has been discussions before (e.g https://github.com/coderefinery/git-intro/issues/80) to portray  it as scary upfront. In-fact if Git did not produce conflicts we will overwrite each others (+our own) contributions.  So my impression of conflict is a a savior not an enemy, as without it happening  things could have been worse. Git is actually providing a   way to handle this, and there is even visual diff possibilities. So I am suggesting to remove the fighting image and then reword this as a good thing as a collaboration.

**Comment by bast** [comment]
Please reword but please keep the image.

**Comment by bast** [comment]
So the point of the images was not to show that conflicts are scary but to have a bit of fun and to avoid a "boring" material. In the text we should definitely try to convince participants that conflicts will happen and that it is normally no problem to resolve them.
